src/Schema/CockroachGrammar.php

```diff
@@ -9,6 +9,23 @@
 
 class CockroachGrammar extends PostgresGrammar
 {
+    /**
+     * Compile the query to determine the tables.
+     *
+     * CockroachDB doesn't yet support pg_total_relation_size()
+     * https://github.com/cockroachdb/cockroach/issues/20712
+     * https://github.com/cockroachdb/cockroach/pull/59604
+     *
+     * @return string
+     */
+    public function compileTables()
+    {
+        return 'select c.relname as name, n.nspname as schema, -1 as size, '
+            . "obj_description(c.oid, 'pg_class') as comment from pg_class c, pg_namespace n "
+            . "where c.relkind = 'r' and n.oid = c.relnamespace "
+            . 'order by c.relname';
```

Comment:
This would be better with consistent quotes

```suggestion
        return 'select c.relname as name, n.nspname as schema, -1 as size, '
            . 'obj_description(c.oid, \'pg_class\') as comment from pg_class c, pg_namespace n '
            . 'where c.relkind = \'r\' and n.oid = c.relnamespace '
            . 'order by c.relname';
```

Reply:
The way I did it is copied directly from Laravel :) I would lean towards inconsistent quotes vs. having to escape characters. Up to you since it's your repo though.

Reply:
A lot of things in the Laravel codebase can be a bit jankie. I'd prefer to improve where possible.